packages/catalog-client/src/schema/openapi/generated/apis/Api.client.ts

```diff
@@ -289,40 +236,115 @@ export class DefaultApiClient {
   }
 
   /**
-   * Delete a location by id.
-   * @param id -
-   */
-  public async deleteLocation(
-    // @ts-ignore
-    request: DeleteLocation,
-    options?: RequestOptions,
-  ): Promise<TypedResponse<void>> {
-    const baseUrl = await this.discoveryApi.getBaseUrl(pluginId);
+     * Get all entities matching a given filter.
+     * @param fields - By default the full entities are returned, but you can pass in a &#x60;fields&#x60; query
+parameter which selects what parts of the entity data to retain. This makes the
+response smaller and faster to transfer, and may allow the catalog to perform
+more efficient queries.
 
-    const uriTemplate = `/locations/{id}`;
+The query parameter value is a comma separated list of simplified JSON paths
+like above. Each path corresponds to the key of either a value, or of a subtree
+root that you want to keep in the output. The rest is pruned away. For example,
+specifying &#x60;?fields&#x3D;metadata.name,metadata.annotations,spec&#x60; retains only the
+&#x60;name&#x60; and &#x60;annotations&#x60; fields of the &#x60;metadata&#x60; of each entity (it&#39;ll be an
+object with at most two keys), keeps the entire &#x60;spec&#x60; unchanged, and cuts out
+all other roots such as &#x60;relations&#x60;.
 
-    const uri = parser.parse(uriTemplate).expand({
-      id: request.path.id,
-    });
+Some more real world usable examples:
 
-    return await this.fetchApi.fetch(`${baseUrl}${uri}`, {
-      headers: {
-        'Content-Type': 'application/json',
-        ...(options?.token && { Authorization: `Bearer ${options?.token}` }),
-      },
-      method: 'DELETE',
-    });
+- Return only enough data to form the full ref of each entity:
+
+  &#x60;/entities/by-query?fields&#x3D;kind,metadata.namespace,metadata.name&#x60;
+
+     * @param limit - Number of records to return in the response.
+     * @param filter - You can pass in one or more filter sets that get matched against each entity.
+Each filter set is a number of conditions that all have to match for the
```

Comment:
is this formatting with just newlines and an outdent safe? If so, I didn't know that